Zone5/Views/ThirdPartyConnectionsView.swift

```diff
@@ -35,16 +36,14 @@ public class ThirdPartyConnectionsView: APIView {
 		return delete(endpoint, with: completion)
 	}
 
-	/// Initialize a connection for the current user for the given 3rd party type
-	/// - Parameters
 	@discardableResult
+	@available(*, deprecated, message: "Use `pairThirdPartyConnection(type:redirect:completion)` instead.")
 	public func initializeThirdPartyConnection(type: UserConnectionType, completion: @escaping Zone5.ResultHandler<ThirdPartyInitializeResponse>) -> PendingRequest? {
 		let endpoint = Endpoints.initializePairing.replacingTokens(["connectionType": type.connectionName])
 		return post(endpoint, body: EmptyBody(), with: completion)
 	}
-	
-	/// Set an access token for the current user for the given 3rd party type
-	/// - Parameters
+
+	/// @available(*, deprecated, message: "Use `pairThirdPartyConnection(type:redirect:completion)` instead.")
```

Comment:
Is this supposed to be uncommented (and thus annotating `setThirdPartyToken(type:parameters:completion:)` below), or did you not intend to leave it in?

Reply:
ha - good catch. I was definitely intending to uncomment that. Thanks!

Reply:
fixed